apps/cli/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import "zx/globals";
+import { parseArgs } from "node:util";
+import { CLI_ARGS_OPTIONS, GLOBAL_CONFIG_FILE } from "./constants";
+import { ensureFileInPath, showHelp } from "./utils";
+import { link } from "./link";
+import type { Commands, SupportedCommands } from "./types";
+
+const commands: SupportedCommands = {
+  link,
+};
+
+export const main = async () => {
+  try {
+    await ensureFileInPath(GLOBAL_CONFIG_FILE, JSON.stringify({}));
+    const args = parseArgs({
+      args: process.argv,
```

Comment:
Usually process.argv.slice(2) is passed to parsers so output never contain system stuff.